lib/tower_cli/commands/config.py

```diff
@@ -146,6 +146,13 @@ def config(key=None, value=None, scope='user', global_=False, unset=False):
         parser.set('general', key, value)
     with open(filename, 'w') as config_file:
         parser.write(config_file)
+        try:
+            os.chmod(filename, int('0600', 8)) # give rw permissions to user only
+                                               # fix for issue number 48
+        except:
+            warnings.warn('Config file permissions can not be edited, which means '
+                          'you could expose your login credentials to other users.',
+                          UserWarning)
```

Comment:
I'd probably simplify this warning message as follow ...

```
Unable to set file permissions on <filename> - <errstr>
```

I'd probably add a call to `warnings.warn()` whenever reading the config file, and `os.stat` reports that the file is group || world readable.